src/Sulu/Component/Content/Types/Link.php

```diff
@@ -71,7 +71,7 @@ public function getViewData(PropertyInterface $property): array
         }
 
         $result = [
-            'provider' => $value['provider'],
+            'provider' => $value['provider'] ?? false,
```

Comment:
```suggestion
            'provider' => $value['provider'] ?? null,
```
This would be consitent to the php default behavior when the key is not set. Otherwise it could be a BC break if stuff is relying on it being null.